initfiles/componentfiles/configxml/buildsetCC.xml.in

```diff
@@ -229,6 +229,10 @@
                         path="WsLogAccess"
                         resource="WsLogAccess"
                         service="ws_logaccess"/>
+                    <AuthenticateFeature description="Access to LogAccess service"
+                        path="WsDfs"
```

Comment:
@jakesmith actually path and description might be wrong here?

Reply:
Yes, I missed those, now fixed.